lib/helpers/has-route.js

```diff
@@ -12,7 +12,7 @@ function isTopLevelExpression(path) {
 }
 
 module.exports = function hasRoute(name, routes, identifier) {
```

Comment:
My main complaint here is that this helper is named `hasRoute` which implies a boolean return. It never was boolean to begin with, but maybe we can fix that at the same time. Change this to `getRoutes`?